src/govuk/helpers/typography.test.js

```diff
@@ -419,4 +417,145 @@ describe('@mixin govuk-typography-responsive', () => {
               line-height: 1.42857; } }`)
     })
   })
+
+  describe('@mixin govuk-font', () => {
+    it('outputs all required typographic CSS properties', async () => {
+      const sass = `
+      // Avoid font face being output in tests
+      $govuk-include-default-font-face: false;
+      ${sassBootstrap}
+
+      .foo {
+        @include govuk-font($size: 14)
+      }`
+
+      const results = await renderSass({ data: sass, ...sassConfig })
+
+      expect(results.css.toString().trim()).toBe(outdent`
+        .foo {
+          font-family: "GDS Transport", arial, sans-serif;
+          -webkit-font-smoothing: antialiased;
+          -moz-osx-font-smoothing: grayscale;
+          font-weight: 400;
+          font-size: 12px;
+          font-size: 0.75rem;
+          line-height: 1.25; }
+          @media print {
+            .foo {
+              font-family: sans-serif; } }
+          @media (min-width: 30em) {
+            .foo {
+              font-size: 14px;
+              font-size: 0.875rem;
+              line-height: 1.42857; } }`)
+    })
+
+    it('enables tabular numbers opentype feature flags if $tabular: true', async () => {
+      const sass = `
+      ${sassBootstrap}
+
+      .foo {
+        @include govuk-font($size: 14, $tabular: true)
+      }`
+
+      const results = await renderSass({ data: sass, ...sassConfig })
+      const css = results.css.toString()
+
+      expect(css).toContain('font-feature-settings: "tnum" 1;')
+      expect(css).toContain(outdent`
+      ${outdent}
+        @supports (font-variant-numeric: tabular-nums) {
+          .foo {
+            font-feature-settings: normal;
+            font-variant-numeric: tabular-nums; } }`)
```

Comment:
Thought it was worth testing for both of these, but not worth writing individual tests for them.